src/cozmo/util.py

```diff
@@ -718,10 +718,45 @@ def q0_q1_q2_q3(self):
 
     @property
     def angle_z(self):
-        '''class:`Angle`: The z Euler component of the object's rotation. Defined as the rotation in the z axis.'''
+        '''class:`Angle`: The z Euler component of the object's rotation.
+            Defined as the rotation in the z axis.
```

Comment:
Docstring formatting / indentation is quite fragile - you need to left align the rest to level with the first quote, and you should generally have a blank line between the 1st line.

See:
https://github.com/anki/cozmo-python-sdk/blob/master/CODESTYLE.md
and example:
http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html